README.md

```diff
@@ -297,6 +297,36 @@ This should produce an installer exe, pkg or rpm+deb file in the `dist/` directo
 
 Building this requires at least 1GB of memory.
 
+#### Apple ARM64 (aka Apple Silicon)
+
+Due to inability to build the management interface proto files on ARM64 (see
+[this](https://github.com/grpc/grpc-node/issues/1497) issue) the Apple ARM64 build must be done in 2 stages:
+
+1. Build management interface proto files on a non-ARM64 platform
+2. Use the built proto files during the main build by setting the `MI_PROTO_BUILD_DIR` environment variable
+   to the path the proto files
+
+To build the management interface proto files there is a script (execute it on a non-ARM64 platform):
+
+```bash
+./build_mi_proto.sh
```

Comment:
What does `mi` mean in the file/directory names?

Reply:
Management Interface. Happy to rename it if you think it's not explicit enough or confusing :)

Reply:
Ah I should have been able to guess that. No need to rename, `mi` works :)

Reply:
Okay, after some internal discussion we came to the conclusion that it would be a good idea to rename variables, scripts and dirs to not include `mi` since it's inconsistent with other shorter versions of `management-interface`. How about naming the script `prepare-proto.sh` or `export-proto.sh` to make it easier to understand how it differs from `build-proto.sh`, and maybe spell out `management interface` in variables and directory names?

Reply:
Now that I look at both `build-proto.sh` and `build_mi_proto.sh`, I think that perhaps it does not make sense to split them. I could do the copying in the `else` clause of the `build-proto.sh` and get rid of `build_mi_proto.sh` completely. So the `else` clause would look like this:

```bash
  BUILD_DIR="$SCRIPT_DIR/management_interface_proto"
  mkdir -p "$BUILD_DIR"

  <run grpc_tools_node_protoc>

  cp $SCRIPT_DIR/../../gui/src/main/management_interface/* $BUILD_DIR
  cp $SCRIPT_DIR/../../gui/build/src/main/management_interface/* $BUILD_DIR

  echo "Management interface proto files built successfully."
  echo "Build directory: $BUILD_DIR"
```

The user would have to perform the build in two steps (after `cd gui/scripts`): 1) `npm ci`; 2) `./build-proto.sh`.
Yes, the copying part would be redundant during the "normal" build, but we will have only one script instead of two.
Another option is no copying at all, and ask the user to collect the proto files from the two `management_interface/` directories themselves.

What do you think?

Reply:
I think I prefer the last option, to not do any copying, as long as the locations are added to the readme. Hopefully `grpc-tools` will add support for the M1 macs in a not to distant future so hopefully this action won't be needed for too long.